common/src/main/java/com/datatorrent/common/util/FSStorageAgent.java

```diff
@@ -153,6 +154,16 @@ public void delete(int operatorId, long windowId) throws IOException
   public long[] getWindowIds(int operatorId) throws IOException
   {
     Path lPath = new Path(path + Path.SEPARATOR + String.valueOf(operatorId));
+    try {
```

Comment:
Seems to be unrelated to the main intent of this PR. Is this a fix for a different bug?